.circleci/config.yml

```diff
@@ -50,10 +50,15 @@ jobs:
     steps:
       - attach_workspace:
           at: /workspace
+      - run:
+          name: Set Environment Variables
```

Comment:
This is local to the shell that sets the variables, and doesn't "reach" the ramp commands. So, if the ramp works, you can remove these entirely.